_thm/Q1033910.md

```diff
@@ -5,4 +5,36 @@ wikidata: Q1033910
 msc_classification: "03"
 wikipedia_links:
   - "[[Cantor–Bernstein–Schroeder theorem]]"
+lean:
+  - status: formalized
+    library: L
+    url: "https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Cardinal/SchroederBernstein.html#Function.Embedding.schroeder_bernstein"
```

Comment:
Note that the Lean entry uses a different URL now (which is independent of the declaration name). Can you make sure to use the "new" URL when merging? Thanks!